### PR TITLE
[test][js-api] Fix import of table64 and memory64

### PR DESCRIPTION
**Pull request by backes**
Provide imports for 'table64' and 'memory64'. This fixes a js-api test failure.

### CODE REVIEW

File: interpreter/script/js.ml
Comment (bvisness) [inline interpreter/script/js.ml]: `initial` and `maximum` should be BigInts (`10n` and `20n`) now that #75 has been merged. Same for the memory you added below.

**Review by bvisness — CHANGES_REQUESTED** [review]

File: interpreter/script/js.ml
Comment (backes) [reply interpreter/script/js.ml]: Ah, yes, that's a change that I still need to do in V8. Thanks for flagging. I'll first add support for BigInt in V8 and then update this PR.

Comment (sbc100) [reply interpreter/script/js.ml]: Is there any reason not to land this now?  I was hoping to get some tests added for #75 assuming they could land here and then v8 could pull them in once its is ready?

Comment (backes) [reply interpreter/script/js.ml]: Implementing BigInt support in V8 first would allow to test this PR in V8 before landing it.

Comment (sbc100) [reply interpreter/script/js.ml]: I'll defer to you here since you are the one doing the work.

However, isn't that backwards from the normal way of doing things.   Don't we normally update the spec + spec tests and then update the implementations once those changes land and get pulled into the testsuite repo?

**Review by sbc100 — APPROVED** [review]

File: interpreter/script/js.ml
Comment (backes) [reply interpreter/script/js.ml]: After the change to BigInt landed now in V8, I changed this PR accordingly and verified that it fixes test failures. Feel free to merge if you agree with the change.

**Pull request merged**
